### DISCUSSION TITLE
Posting a question in an existing discussion

### DISCUSSION THREAD

**Discussion by teztesla** [aclap-dev/video-downloadhelper#795] [Help and Q&A]
### Your question

New to GH, want to post a question in an existing discussion, I cant see any buttons or hotlinks to 'add question/post' in the discussion page of a group.
Call me a dummy as I have missed it.

### Addon & CoApp version and details

DLH beta for firefox on mac

**Comment by jcfrog** [comment]
Hi,
Don't you have a "Suggest answer" field like on this post?
<img width="942" alt="image" src="https://github.com/aclap-dev/video-downloadhelper/assets/368367/d780c850-7a17-4e3e-8f36-8e2478047f52">

**Reply by teztesla** [reply]
Thank you JCFROG, I can see the Suggest Answer when within a discussion, what I was wanting to do is start a new discussion within the group. And, yay, I have now worked this out.
Thank you for your reply.
